Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "near-lake"
-version = "0.2.0-rc.1"
+version = "0.1.0"
```

Comment:
Is it ok that we create the next version with the decreased version number?

Reply:
we didn't actually release the 0.1.0, so why not? :)